admin-tools/get-credentials.yaml

```
@@ -107,7 +107,7 @@
         kind: Secret
```

Dans le nom de la tâche :
```suggestion
      - name: Get DSO config Secret from DSO console namespace
```